test/MyFirstApiConsume.test.js

```diff
@@ -0,0 +1,52 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+describe('First Api Tests', () => {
+    it('Consume GET Service', async () => {
+        const response = await axios.get('https://httpbin.org/ip');
+      
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.data).to.have.property('origin');
+    });
+
+    it('Consume GET Service with query parameters', async () => {
+        const query = {
+            name: 'John',
+            age: '31',
+            city: 'New York'
+        };
+        
+        const response = await axios.get('https://httpbin.org/get', { query });
+        
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.config.query).to.eql(query);
+    });
+
+    it('Consume HEAD Service', async () => {
+        const response = await axios.head('https://httpbin.org/get',);
+      
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(parseInt(response.headers['content-length'])).to.be.greaterThan(0);
+    });
+
+    it('Consume PATCH Service', async () => {
+        const response = await axios.patch('https://httpbin.org/patch');
+      
+        expect(response.status).to.equal(StatusCodes.OK);
+    });
+
+    it('Consume PUT Service', async () => {
+        const query = { name: 'Joe' };
```

Comment:
rename query variable to data, payload or body, since in this case the object associated to the variable is being sent as payload instead of query params

Reply:
Understood